.github/workflows/main.yml

```diff
@@ -255,63 +255,3 @@ jobs:
         with:
           path-to-profile: coverage.txt
           parallel: true
-
```

Comment:
Not in the diff part but the GH workflow still runs `install_bitcoind.sh`, is that still needed?

Reply:
Good eye for details, but unfortunately `bitcoind` is required for some of the unit tests.

Reply:
Good to know, thanks :)